src/main/java/com/okta/sdk/clients/UserApiClient.java

```diff
@@ -78,6 +78,9 @@ public UserApiClient(ApiClientConfiguration config) {
      * @throws IOException  If an input or output exception occurred.
      */
     public List<User> getUsersWithLimit(int limit) throws IOException {
+        if (limit <= 0) {
```

Comment:
The API will return users when `limit` is `-1`. Anything `< -1` throws a `400`. Should we just check the zero case and let Okta return the error message?